modules/ocf_kube/manifests/master.pp

```diff
@@ -0,0 +1,218 @@
+      content_path => "${certs_dir}/apiserver.crt";
+    '/etc/kubernetes/pki/apiserver.key':
+      content_path => "${certs_dir}/apiserver.key";
+
+    # kubelet server key
+    '/etc/kubernetes/pki/kubelet-server.crt':
+      content_path => "${certs_dir}/${::hostname}-kubelet-server.crt";
+    '/etc/kubernetes/pki/kubelet-server.key':
+      content_path => "${certs_dir}/${::hostname}-kubelet-server.key";
+
+    # kubelet -> apiserver client key
+    '/etc/kubernetes/pki/apiserver-kubelet-client.crt':
+      content_path => "${certs_dir}/apiserver-kubelet-client.crt";
+    '/etc/kubernetes/pki/apiserver-kubelet-client.key':
+      content_path => "${certs_dir}/apiserver-kubelet-client.key";
+
+    # kubernetes ca
+    '/etc/kubernetes/pki/ca.crt':
+      content_path => "${certs_dir}/kube-ca.crt";
+    # don't copy the private key
+
+    # controller-manager -> apiserver client key
+    '/etc/kubernetes/pki/admin.crt':
+      content_path => "${certs_dir}/admin.crt";
+    '/etc/kubernetes/pki/admin.key':
+      content_path => "${certs_dir}/admin.key";
+
+    # controller-manager -> apiserver client key
+    '/etc/kubernetes/pki/controller-manager.crt':
+      content_path => "${certs_dir}/controller-manager.crt";
+    '/etc/kubernetes/pki/controller-manager.key':
+      content_path => "${certs_dir}/controller-manager.key";
+
+    # scheduler -> apiserver client key
+    '/etc/kubernetes/pki/scheduler.crt':
+      content_path => "${certs_dir}/scheduler.crt";
+    '/etc/kubernetes/pki/scheduler.key':
+      content_path => "${certs_dir}/scheduler.key";
+
+    # service account keypair
+    '/etc/kubernetes/pki/sa.pub':
+      content_path => "${certs_dir}/service.pub";
+    '/etc/kubernetes/pki/sa.key':
+      content_path => "${certs_dir}/service.key";
+
+    # etcd ca
+    '/etc/kubernetes/pki/etcd/ca.crt':
+      content_path => "${certs_dir}/etcd-ca.crt";
+    # don't copy the private key
+
+    # apiserver/etcd -> etcd client key
+    '/etc/kubernetes/pki/etcd/client.crt':
+      content_path => "${certs_dir}/${::hostname}-etcd-client.crt";
+    '/etc/kubernetes/pki/etcd/client.key':
+      content_path => "${certs_dir}/${::hostname}-etcd-client.key";
+
+    # etcd server key
+    '/etc/kubernetes/pki/etcd/server.crt':
+      content_path => "${certs_dir}/${::hostname}-etcd-server.crt";
+    '/etc/kubernetes/pki/etcd/server.key':
+      content_path => "${certs_dir}/${::hostname}-etcd-server.key";
+
+    # front-proxy ca
+    '/etc/kubernetes/pki/front-proxy-ca.crt':
+      content_path => "${certs_dir}/front-proxy-ca.crt";
+    '/etc/kubernetes/pki/front-proxy-ca.key':
+      content_path => "${certs_dir}/front-proxy-ca.key";
+
+    # front-proxy client
+    '/etc/kubernetes/pki/front-proxy-client.crt':
+      content_path => "${certs_dir}/front-proxy-client.crt";
+    '/etc/kubernetes/pki/front-proxy-client.key':
+      content_path => "${certs_dir}/front-proxy-client.key";
+  }
+
+  # TODO: disable swap
```

Comment:
`swapoff` based on `swapon -s` should work?

Reply:
You would also want to comment it out in /etc/fstab, otherwise you need to wait for puppet to run, disabling swap before kubelet will even start. That's certainly possible to do in puppet, but I'll leave it for a later PR.